sql/moz-fx-data-shared-prod/firefox_accounts_derived/health_counts_v1/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+friendly_name: Accounts Health Counts
+description: >
+  Counts calculated daily regarding the health of the Accounts service.  E.g. # of issues open.
+owners:
+  - wclouser@mozilla.com
+external_data:
+  format: google_sheets
+  source_uris:
+    - https://docs.google.com/spreadsheets/d/1uGHOS-cY_OUqicbBkS2Z20EK0hLq3PDw28TZcxP5RHI/edit?gid=1783447085 # URL to the spreadsheet
```

Comment:
I'd suggest removing `?gid=1783447085` from all these URLs, as it's slightly misleading since that `gid` is specifically for the "Lighthouse" tab (BigQuery ignores that query parameter anyway).